config/webpack.common.js

```diff
@@ -287,7 +286,6 @@ module.exports = function (webpackEnv, args) {
                 tsBuildInfoFile: paths.misc.tsBuildInfoFile,
               },
             },
-            context: paths.sources.srcFolder,
```

Comment:
We discussed this to stay in place as we do not want to lint third party packages. We do want to add an exclude for `node_modules` for the same reason.

Reply:
Done